backend-rust/src/bin/ccdscan-indexer.rs

```diff
@@ -83,7 +84,17 @@ async fn main() -> anyhow::Result<()> {
         let _ = dotenvy::dotenv();
     }
     let cli = Cli::parse();
-    tracing_subscriber::fmt().with_max_level(cli.log_level).init();
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::fmt::layer())
+        .with(
+            tracing_subscriber::EnvFilter::builder()
+                .with_default_directive(
+                    format!("{}={}", env!("CARGO_PKG_NAME").replace('-', "_"), cli.log_level)
```

Comment:
can we add a comment here. People might not directly understand what we filter  out here and why?